### PR TITLE
[KFI] Zip file always unpacked.

### PR DESCRIPTION
**Pull request by kavics**
Please review and merge.

### CODE REVIEW

**Comment by kavics** [comment]
Manual test cases are successfully executed.

"pkg": a directory that contains manifest.xml with a Trace command: "overridden message"
"pkg.zip": compressed package with a Trace command: "original message"

Worklog:
```
	before		after		SnAdmin
Case	pkg.zip	pkg	pkg.zip	pkg	command		Expected message		Result
===============================================================================================
1	-	-	-	-	pkg		Given package does not exist	ok
2	-	-	-	-	pkg.zip		Given package does not exist	ok
3	-	exists	-	exists	pkg		overridden message		ok
4	-	exists	-	exists	pkg.zip		Given package does not exist	ok
5	exists	-	exists	exists	pkg		original message		ok
6	exists	-	exists	exists	pkg.zip		original message		ok
7	exists	exists	exists	exists	pkg		original message		ok
8	exists	exists	exists	exists	pkg.zip		original message		ok
```

File: src/SnAdmin/SnAdmin.cs [hunk 1]
```diff
@@ -193,10 +193,10 @@ private static int ExecuteGlobal(string packagePath, string sandboxDirectory, st
             int result;
             var phase = 0;
             var errors = 0;
-            string workerExe;
 
             while (true)
             {
+                string workerExe;
```
Comment (tusmester) [inline src/SnAdmin/SnAdmin.cs]: Why is this needed?

File: src/SnAdminTests/ExecutionTests.cs [hunk 2]
```diff
@@ -30,7 +32,7 @@ public void Execution_OnePhase()
             var result = SnAdmin.Main(args);
 
             // ASSERT
-            var consoleText = console.GetStringBuilder().ToString();
+            //var consoleText = console.GetStringBuilder().ToString();
```
Comment (tusmester) [inline src/SnAdminTests/ExecutionTests.cs]: Remove or rewrite pls.

File: src/SnAdminTests/ExecutionTests.cs [hunk 3]
```diff
@@ -53,7 +55,7 @@ public void Execution_ThreePhases()
...
```
Comment (tusmester) [inline src/SnAdminTests/ExecutionTests.cs]: Remove or rewrite pls.

**Review by tusmester — CHANGES_REQUESTED** [review]

**Review by tusmester — APPROVED** [review]

**Pull request merged**